### ISSUE TITLE
Documentation Update ?

### ISSUE THREAD

**Issue by rohitsangwan01** [asadbaidar/get_smart#1]
Hey , i was just going through your package , and there are lots of really usefull methods , can u add a documentation , i was planning to add this to my projects as well

**Comment by asadbaidar** [comment]
> Hey , i was just going through your package , and there are lots of really usefull methods , can u add a documentation , i was planning to add this to my projects as well

Yeah I'm working on it but due to time shortage it's getting delayed. For the time being, let me know how can I help you understanding the package.

**Comment by rohitsangwan01** [comment]
Great, till then can you list down few mostly used Methods overhere ,or in Docs maybe as a Temp. Documetation

**Comment by asadbaidar** [comment]
> Great, till then can you list down few mostly used Methods overhere ,or in Docs maybe as a Temp. Documetation

Sure
